docs/source/adrs/0012-concourse-teams.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 0012 Concourse Teams for CI and Deploy 
+hide_in_navigation: true
+---
+
+# ADR 12: Concourse Teams for CI and Deploy
+
+Date: 03-08-20
+
+## Status
+
+Accepted
+
+## Context
+
+Currently GOV.UK Pay has one team for both the CI and Deploy portions or our Concourse CI environment, this is problematic as too many concerns are grouped within one team and GDS best practice is to separate these concerns where possible. Our current model in Jenkins is a multi environment solution for both CI and Deploy and it makes sense for the new architecture to match that. This will allow us to greatly simplify our Concourse environment which currently is quite cluttered because it is required to do so much within a single team.
+
+Our proposal is to move the CI pipelines and related infrastructure to a new team, ```pay-ci``` and keep the current deploy components and pipelines within the ```pay-deploy``` team.
```

Comment:
I've already done this and called the team `pay-dev` as is RE convention: https://reliability-engineering.cloudapps.digital/continuous-deployment.html#teams

Reply:
sound - will fix